nixos/modules/tasks/network-interfaces-scripted.nix

```
@@ -200,14 +200,16 @@ let
                 ${flip concatMapStrings ips (ip:
                   let
                     cidr = "${ip.address}/${toString ip.prefixLength}";
+                    optionalPeer = lib.optionalString (ip.peer != null) (" " + ''peer "${ip.peer}"'');
+                    optionalPeerForEcho = lib.optionalString (ip.peer != null) (" " + ''peer \"${ip.peer}\"'');
```

I know, I'm being nitpicky, but mixing `''` and `"` look very ugly and hard to read:

```suggestion
                    optionalPeer = lib.optionalString (ip.peer != null) (" peer \"${ip.peer}\"");
```

And later for echo:

```sh
echo ${lib.escapeShellArg "'ip addr replace \"${cidr}\"${optionalPeer} dev \"${i.name}\"' failed:"} "$out"
```

Of course, we could also just use `lib.escapeShellArg` everywhere, even if that would mean that I can no longer use hacks like eg. setting `address` to `1.2.3.4/16" dev "foo"\n...whatever i want...\n#`.